DependencyInjection/Configuration.php

```diff
@@ -111,6 +111,7 @@ public function getConfigTreeBuilder()
                                         ->integerNode('port')->defaultValue(6379)->end()
                                         ->scalarNode('host')->defaultValue('localhost')->end()
                                         ->integerNode('database')->isRequired()->end()
+                                        ->integerNode('password')->defaultValue(null)->end()
```

Comment:
You're totally right. I was not quite alert this morning. Password is a scalarNode now. Database was always an integerNode, so no change there. I didn't pay attention to the detail when editing the pull request and kept the integerNode on the password key, so apologies there.

The rest of the code works without modification as the SonataCacheBundle sends the whole Predis configuration array as connection information to Predis, which allows for a password key.